doc/glossary.rst

```diff
@@ -75,19 +75,19 @@
       can be scripted with a custom GUI / batch system, or using shell
       scripting.  Its particular strength is analysis of :term:`DWI`
       data, and :term:`ICA` functional data analysis, although it has
-      strong tools for the standard :term:`SPM approach` to FMRI. It
+      strong tools for the standard :term:`SPM approach` to FMRI.  It
       is free for academic use, and open-source, but not free for
       commercial use.
 
-   GPL 
+   GPL
       The GPL_ is the GNU general public license.  It is one of the
-      most commonly-used open-source sofware licenses.  The
+      most commonly-used open-source software licenses.  The
       distinctive feature of the GPL license is that it requires that
       any code derived from GPL code also uses a GPL license.  It also
       requires that any code that is statically or dynamically linked
-      to GPL code has a GPL-compatible license. See:
-      http://en.wikipedia.org/wiki/GNU_General_Public_License and
-      http://www.gnu.org/licenses/gpl-faq.html
+      to GPL code has a GPL-compatible license.  See: `Wikipedia GPL
```

Comment:
Could these already be in the general `links_names` file included at the end of this file?